src/acquisition/fluview/fluview_update.py

```diff
@@ -355,12 +399,13 @@ def update_from_file_public(issue, date, filename, test_mode=False):
   # insert each row
   insert = cnx.cursor()
   for row in entries:
+    lag = delta_epiweeks(row['epiweek'], issue)
```

Comment:
Looks like `lag` won't be NULL/None, but the comment above says it'll be NULL for state data. Either way is fine with me (making `lag` `NOT NULL` in database, or setting it to `None` here).

Reply:
I decided to set it to `NOT NULL` in the database! Thanks for all the comments, I really appreciate it!